assets/admin/students/student-modal/index.js

```diff
@@ -110,7 +132,7 @@ export const StudentModal = ( { action, onClose, students } ) => {
 			title={ __( 'Choose Course', 'sensei-lms' ) }
 			onRequestClose={ () => onClose() }
 		>
-			<p>{ description }</p>
+			<RawHTML>{ replacedDescription }</RawHTML>
```

Comment:
This value actually comes from the action menu markup, it's escaped before rendering. Also, the name is sanitized before saving. I was a bit concerned as well before writing this, but I think given the measures taken for it in those places, it will be alright. If anyone tampers in the client-side, it'll stay there.

Reply:
Is it vulnerable to XSS?

Reply:
I wouldn't agree we have 100% control over what will come in the student's name and also where and how will this component be used in some other place

Reply:
Right, thanks. I've added an update [here](https://github.com/Automattic/sensei/pull/5038/commits/b37dcd23801a8c7f696af69ffa0b4a12d0a744a0), can you take a look again?